hoot-js/src/main/cpp/hoot/js/conflate/ElementMergerJs.cpp

```diff
@@ -0,0 +1,264 @@
+/*
```

Comment:
This is the main class for the merging core changes.  Its a central point where you can pass any set of features and hoot will merge them if it can.